src/panels/config/backup/components/ha-backup-agents-select.ts

```diff
@@ -0,0 +1,104 @@
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import "../../../../components/ha-checkbox";
+import "../../../../components/ha-formfield";
+import type { BackupAgent } from "../../../../data/backup";
+import type { HomeAssistant } from "../../../../types";
+import { brandsUrl } from "../../../../util/brands-url";
+
+@customElement("ha-backup-agent-select")
+class HaBackupAgentsSelect extends LitElement {
+  @property({ attribute: false })
+  public hass!: HomeAssistant;
+
+  @property({ type: Boolean })
+  public disabled = false;
+
+  @property()
+  public agents!: BackupAgent[];
+
+  @property({ attribute: false })
+  public disabledAgents?: string[];
+
+  @property()
+  public value!: string[];
+
+  render() {
+    return html`
+      <div class="agents">
+        ${this.agents.map((agent) => this._renderAgent(agent))}
+      </div>
+    `;
+  }
+
+  private _renderAgent(agent: BackupAgent) {
+    const [domain, name] = agent.agent_id.split(".");
+    const domainName =
```

Comment:
Use the `domainToName` helper function

Reply:
It's not really a domain like entity domain. I'm also not sure this format will stay.